pandas/tests/frame/test_dtypes.py

```diff
@@ -523,6 +523,24 @@ def test_timedeltas(self):
         result = df.get_dtype_counts().sort_values()
         assert_series_equal(result, expected)
 
+    def test_illegal_arg_for_errors_in_astype(self):
```

Comment:
you can make this a single test.

as the same test in tests/series/test_dtypes.py as well